src/dbcp/data_mart/__init__.py

```diff
@@ -4,83 +4,19 @@
 import logging
 import pkgutil
 
-import pandas as pd
-import pyarrow as pa
-import pyarrow.parquet as pq
-
-import dbcp
-from dbcp.constants import OUTPUT_DIR
-from dbcp.helpers import enforce_dtypes, psql_insert_copy
-from dbcp.metadata.data_mart import metadata
-from dbcp.validation.tests import validate_data_mart
-
 logger = logging.getLogger(__name__)
 
 
-def create_data_marts():  # noqa: max-complexity=11
-    """Collect and load all data mart tables to data warehouse."""
-    engine = dbcp.helpers.get_sql_engine()
-    data_marts = {}
+def get_data_mart_modules():
+    """Get the modules in the data mart directory in prep for creating data mart tables."""
     modules_to_skip = {
         "helpers",  # helper code; no tables
         "co2_dashboard",  # obsolete but code imported elsewhere
     }
-
+    modules = []
     for module_info in pkgutil.iter_modules(__path__):
         if module_info.name in modules_to_skip:
             continue
         module = importlib.import_module(f"{__name__}.{module_info.name}")
-        try:
```

Comment:
I moved all of the actual table creation to `create_data_mart` in `etl.py` to be more consistent with how the data warehouse tables are created.